ranker/ranker.py

```diff
@@ -57,6 +58,7 @@ def rankQuery(query):
     pureQuery = query[0:index]
     
   queryTerms = stemQuery(pureQuery, stopwords)
+  addSpecialChars(queryTerms)
```

Comment:
Output needs to be stored.
You can do queryTerms = addSpecialChars(stemQuery(pureQuery, stopwords))